p8_integration_tests/quick_test/test_spike_source/test_spike_source_array.py

```diff
@@ -132,7 +132,7 @@ def test_recording_numerous_element_with_zero(self):
     def recording_with_empty_lists(self):
         p.setup(timestep=1.0)
         p.set_number_of_neurons_per_core(p.SpikeSourceArray, 2)
-        spike_times = [[1], [], [], [], [4], [3]]
+        spike_times = [[], [1], [], [], [4], [3]]
```

Comment:
why did you do this change? is there something special about the first array?

Reply:
yes. The check if spike_times are a single list or multiple looks at the first array. So wanted to make sure it worked even if first array is empty.

Reply:
do we have a second test that does the initial version, in case we break it the other way?

Reply:
It would be potentially easy to do that as part of this test too i.e. have one one way and one the other.